src/main/level_file.cpp

```diff
@@ -199,6 +199,25 @@ bool OpenLevelData(LevelData &lvl, const std::string FilePath)
 
     IsEpisodeIntro = (StartLevel == FileNameFull);
 
+    if(IsEpisodeIntro)
+    {
+        IsHubLevel = NoMap;
+        FileRecentSubHubLevel.clear();
+    }
+
+    // Level-wide extra settings
+    if(!lvl.custom_params.empty())
+    {
+        const nlohmann::json level_data = nlohmann::json::parse(lvl.custom_params);
```

Comment:
Please check for exceptions here (the `try` block should also include the `value` check, that throws if there is a type mismatch)

Reply:
Ouch, forgot, good catch!